_config.yml

```diff
@@ -47,7 +47,7 @@
       },
       "links": {
         "en": [{
-          "url": "mailto:cds.dto-btn.snc@servicecanada.gc.ca",
+          "url": "https://blog.canada.ca/contact-us",
```

Comment:
URL doesn't seem to be there yet, but maybe that's a different PR

Reply:
Indeed, I'm merging that PR right now.

Reply:
The link is live now